xbmc/interfaces/json-rpc/schema/types.json

```diff
@@ -1476,7 +1476,9 @@
       "displaycomposer": { "type": "string"},
       "displayconductor": { "type": "string"},
       "displayorchestra": { "type": "string"},
-      "displaylyricist": { "type": "string"}
+      "displaylyricist": { "type": "string"},
+      "mediapath": { "type": "string", "description": "Media source path that identifies the item"},
+      "dynpath": { "type": "string", "description": "An experimental property for debug purposes, often same as mediapath but when different gives the actual file playing that should also be in file property"}
```

Comment:
The description is wrong. dynpath is neither experimental nor a debug property. Sorry, but have not spotted this earlier.